templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -29,9 +29,8 @@
 
     inputfield.value = value;
     }
-    [% INCLUDE web/pages/user_form/togglepassword.tt.js %]
 </script>
-
+<script type="text/javascript" src="/js/togglepassword.js"></script>
```

Comment:
Could you add both JS scripts to [gulp](https://github.com/openfoodfacts/openfoodfacts-server/blob/34a46b891c1a7195a5d424cc991d309c8f269b31/gulpfile.ts#L13), so that they're properly minified for deployment. You'll have to load them from `/js/dist/togglepassword.js` to use that effect.

Reply:
We also sometimes use `static_subdomain` to ensure the JS files are loaded from a canonical URL: https://github.com/openfoodfacts/openfoodfacts-server/blob/34a46b891c1a7195a5d424cc991d309c8f269b31/templates/web/common/site_layout.tt.html#L409

Reply:
Hi @hangy, yes sure, added now👍
Thanks!